readme.md

```diff
@@ -162,7 +207,7 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 </html>
 ```
 
-> Your answer...
+> title tag needs to be closed
 
```

Comment:
While this is true, the more important thing to fix is the `yield` statement. If an ERB statement is going to render anything on the page, it needs to include an `=`. So it should look something like this...

```ruby
<%= yield %>
```